sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -3320,6 +3320,10 @@ _ZNK2cl4sycl13host_selectorclERKNS0_6deviceE
 _ZNK2cl4sycl14exception_list3endEv
```

Comment:
@s-kanaev, @alexbatashev, I think we should add a comment to this test saying that together with this test API version must be updated in sycl/CMakeLists.txt. Right?

Reply:
In this particular case no version changes are needed. I'll modify the script to emit some link to ABI Policy Guide.